.swiftlint.yml

```diff
@@ -6,6 +6,12 @@ excluded:
 opt_in_rules:
   - empty_count
   - file_header
+  - explicit_init
+  - closure_spacing
+  - overridden_super_call
+  - redundant_nil_coalescing
+  - private_outlet
+  - nimble_operator
```

Comment:
I've commented about this on the PR body, but the idea is to add an extra level of integration test, to make sure these rules aren't being trigger where they shouldn't. This applies to the `private_outlet` rule as well.

As they only add 0,012s, I thought it was useful to add this sanity check.